cura/CuraActions.py

```diff
@@ -181,5 +187,63 @@ def setBuildPlateForSelection(self, build_plate_nr: int) -> None:
 
         Selection.clear()
 
+    @pyqtSlot()
+    def cut(self) -> None:
+        self.copy()
+        self.deleteSelection()
+
+    @pyqtSlot()
+    def copy(self) -> None:
+        # Convert all selected objects to a Savitar scene
+        savitar_scene = Savitar.Scene()
+        mesh_writer = cura.CuraApplication.CuraApplication.getInstance().getMeshFileHandler().getWriter("3MFWriter")
+        for scene_node in Selection.getAllSelectedObjects():
+            savitar_node = mesh_writer._convertUMNodeToSavitarNode(scene_node)
+            savitar_scene.addSceneNode(savitar_node)
+
+        # Convert the scene to a string
+        parser = Savitar.ThreeMFParser()
+        scene_string = parser.sceneToString(savitar_scene)
+
+        # Copy the scene to the clipboard
+        QApplication.clipboard().setText(scene_string)
+
+    @pyqtSlot()
+    def paste(self) -> None:
+        application = cura.CuraApplication.CuraApplication.getInstance()
+
+        # Parse the scene from the clipboard
+        scene_string = QApplication.clipboard().text()
+        parser = Savitar.ThreeMFParser()
+        scene = parser.parse(scene_string)
+
+        # Convert the scene to scene nodes
+        nodes = []
+        mesh_reader = application.getMeshFileHandler().getReaderForFile(".3mf")
+        for savitar_node in scene.getSceneNodes():
```

Comment:
Same as above, but inverted; Just add a function that returns a list of nodes given a string.

Reply:
https://github.com/Ultimaker/Cura/pull/16383/commits/f8b3fb3d67890b5ee3d2aeac945c9ff136cf62bf